wp-content/themes/csisnuclearnetwork/template-parts/content.php

```diff
@@ -23,21 +23,25 @@
 
 	<div class="single__content">
 		<?php
-      the_content( __( 'Continue reading', 'nuclearnetwork' ) );
-      nuclearnetwork_display_footnotes();
-			get_template_part( 'template-parts/featured-image-caption' );
-			nuclearnetwork_citation();
-    ?>
+      		the_content( __( 'Continue reading', 'nuclearnetwork' ) );
+			nuclearnetwork_display_footnotes();
+		?>
 	</div><!-- .post-inner -->
 
 	<?php if ( is_single() ) { ?>
-	<!-- <footer class="single__footer"> -->
+	<footer class="single__footer">
 		<?php
-			// nuclearnetwork_authors_list_extended();
-			// echo do_shortcode( '[jprel]' );
-			// if ( function_exists( 'ADDTOANY_SHARE_SAVE_KIT' ) ) { ADDTOANY_SHARE_SAVE_KIT(); }
+
+		    echo '<div class="post__authors"><hr>';
+			echo '<h2 class="section__heading post__authors-heading">Authors</h2><p class="text--italic text--short post__authors-disclaimer">The views expressed above are the author’s and do not necessarily reflect those of the Center for Strategic and International Studies or the Project on Nuclear Issues.</p><div class="post__authors-group">';
+			nuclearnetwork_authors_list_extended(); 
+			echo '</div><div class="single__footer__write-for-us">';
+			dynamic_sidebar('write-for-us'); 
+			echo '</div></div>';
+			nuclearnetwork_citation();
+			// get_template_part( 'template-parts/featured-image-caption' );
```

Comment:
The `jprel` shortcode and share buttons shouldn't have been removed.

`// echo do_shortcode( '[jprel]' );`
`// if ( function_exists( 'ADDTOANY_SHARE_SAVE_KIT' ) ) { ADDTOANY_SHARE_SAVE_KIT(); }`